templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -186,7 +186,7 @@ page. If you haven't created a bot already, you can do that there. Then copy
 its API key and replace the placeholder `<api_key>` in the examples with
 your real key. This is how Zulip knows the request is from an authorized user.
 
-Now you can test using Zulip itself, or curl on the command line.
+Now you can test using Zulip itself (either via. command line or GUI), or curl on the command line.
```

Comment:
Hmm.. this line could be worded a bit better. Maybe something like `You can test incoming webhooks via curl or the Integrations Developer Panel (GUI).`

Reply:
Ok, I'll use this exact line :)